assets/sass/components/global/_googlesitekit-entity-header.scss

```diff
@@ -93,33 +99,28 @@
 
 		p {
 			font-family: $f-secondary;
-			font-size: 11px;
-			line-height: 14px;
 			margin: 0;
 			overflow: hidden;
 			text-overflow: ellipsis;
 
+			@include responsive-text("body3", "body3");
```

Comment:
Not a problem per-se, but if we agree on https://github.com/google/site-kit-wp/issues/4428#issuecomment-1010995527, this would need to also use `body2` for the second value, similar to the CSS for the "Back to dashboard" button above.